Library/Homebrew/rubocops/uses_from_macos.rb

```diff
@@ -98,7 +98,12 @@ class UsesFromMacos < FormulaCop
         def audit_formula(_node, _class_node, _parent_class_node, body_node)
           return if body_node.nil?
 
-          find_method_with_args(body_node, :uses_from_macos, /^"(.+)"/).each do |method|
+          uses_from_macos_nodes = find_method_with_args(body_node, :uses_from_macos, /^"(.+)"/)
+          return if uses_from_macos_nodes.blank?
+
+          problem "`uses_from_macos` should not be used when Linux is required." if depends_on?(:linux)
```

Comment:
I will need to run `depends_on?` first to avoid `@offensive_node` being set to `depends_on :linux`.

On side note, previous logic is off as `find_method_with_args` is finding every method beforehand so `@offensive_node` is always the last match (so it may point to a "correct" usage rather that actual problem). 

Easiest fix is probably `@offensive_node = method`.